pkg/virt-handler/vm.go

```diff
@@ -21,6 +22,18 @@ func NewVMController(listWatcher cache.ListerWatcher, domainManager libvirt.Doma
 				recorder.Event(vm, kubev1.EventTypeWarning, v1.SyncFailed.String(), err.Error())
 				return cache.ErrRequeue{Err: err}
 			}
+
+			restClient, err := kubecli.GetRESTClient()
+			if err != nil {
+				return cache.ErrRequeue{Err: err}
+			}
+			vm.Status.Phase = v1.Succeeded
```

Comment:
`Succeeded` in the kubernetes world means that a Pod/Container/... stopped or was stopped without errors.
We need it to go to `Running` state to keep the kubernetes sematics.

Examples for going into `Succeeded`:
 * User shut down the VM and it was stopped successfully
 * (Maybe) graceful delete

Examples for `Failed`:
 * Can't start the VM
 * VM crashed